hydromt/models/root.py

```diff
@@ -58,6 +86,33 @@ def is_reading_mode(self) -> bool:
         """Test whether we are in reading mode or not."""
         return self._mode.is_reading_mode()
 
+    # old_root is so we can copy over the old log file if needed.
+    def _create_loggers(self, old_path: Optional[Path] = None, overwrite: bool = False):
```

Comment:
For me it would be more clear to change the name into something like `_update_logger_filehandler` as it does not create a new logger instance.

Also, could you add a one line docstring to clarify what the method does. Something like
"""Update the file handler to save logs in self.path/hydromt.log. If a log file at old_path exists copy it to self.path and append."""